pkg/config/v3/config.go

```diff
@@ -234,7 +234,7 @@ func (c *Cfg) UpdateResource(res resource.Resource) error {
 	}
 
 	for i, r := range c.Resources {
-		if res.GVK.IsEqualTo(r.GVK) {
+		if res.IsEqualTo(r.GVK) {
```

Comment:
will not it change the behaviour?

Reply:
Thanks @camilamacedo86 
Yes. `GVK` just inline parameters, fixed by `golangci-lint run --fix`

Reply:
I am not sure if: `res.IsEqualTo()` is just forwarding to `res.GVK.IsEqualTo()`
So, I have some concerns about it. I will need to check it further.
OR could we not keep it as before?

Reply:
Checked it is fine 👍